src/js/player.js

```diff
@@ -4290,6 +4290,35 @@ class Player extends Component {
     return !!this.isAudio_;
   }
 
+  /**
+   * Get the current audioPosterMode state or set audioPosterMode to true or false
+   *
+   * @param {boolean} [value]
+   *         The value to set audioPosterMode to.
+   *
+   * @return {boolean}
+   *         True if audioPosterMode is on, false otherwise.
+   */
+  audioPosterMode(value) {
+
+    value = value && Boolean(value);
+
+    if (value === undefined || value === this.options_.audioPosterMode) {
+      return this.options_.audioPosterMode;
+    }
+
+    this.options_.audioPosterMode = value;
+
+    if (this.options_.audioPosterMode) {
+      this.tech_.hide();
```

Comment:
thought: hiding the tech explicitly could potentially cause playback issues, particularly with other techs like youtube. Though, I'm not positive. Probably not worth handling now, but might need another solution if we do run into such a problem in the future.